diff --git a/debian/cosmic-greeter.pam b/debian/cosmic-greeter.pam
index 2a15d02..7c0a040 100644
--- a/debian/cosmic-greeter.pam
+++ b/debian/cosmic-greeter.pam
@@ -19,7 +19,6 @@ session [success=ok ignore=ignore module_unknown=ignore default=bad]        pam_
 session optional        pam_keyinit.so force revoke
 session required        pam_limits.so
 session required        pam_env.so readenv=1
-session required        pam_env.so readenv=1 user_readenv=1 envfile=/etc/default/locale
 @include common-session
 session optional        pam_gnome_keyring.so auto_start
 @include common-password
