diff --git a/configure.ac b/configure.ac
index 22fee70..88e24ad 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2842,7 +2842,7 @@ if test "x$openssl" = "xyes" ; then
 				# OpenSSL 3; we use the 1.1x API
 				CPPFLAGS="$CPPFLAGS -DOPENSSL_API_COMPAT=0x10100000L"
 				;;
-			301*|302*)
+			301*|302*|303*)
 				# OpenSSL development branch; request 1.1x API
 				CPPFLAGS="$CPPFLAGS -DOPENSSL_API_COMPAT=0x10100000L"
 				;;
