diff --git a/cosmic-settings/src/pages/system/users/mod.rs b/cosmic-settings/src/pages/system/users/mod.rs
index ce17914..b092f8c 100644
--- a/cosmic-settings/src/pages/system/users/mod.rs
+++ b/cosmic-settings/src/pages/system/users/mod.rs
@@ -252,7 +252,7 @@ impl Page {
 
         let uid = rustix::process::getuid().as_raw() as u64;
 
-        let admin_group = groups.iter().find(|g| &*g.name == "sudo");
+        let admin_group = groups.iter().find(|g| &*g.name == "wheel");
 
         let Ok(conn) = zbus::Connection::system().await else {
             tracing::error!("unable to access dbus system service");
diff --git a/resources/polkit-1/rules.d/cosmic-settings.rules b/resources/polkit-1/rules.d/cosmic-settings.rules
index 54a1eb0..46f8a60 100644
--- a/resources/polkit-1/rules.d/cosmic-settings.rules
+++ b/resources/polkit-1/rules.d/cosmic-settings.rules
@@ -6,7 +6,7 @@ polkit.addRule(function(action, subject) {
             action.id == "org.freedesktop.hostname1.set-hostname") &&
         subject.local &&
         subject.active &&
-        subject.isInGroup ("sudo")) {
+        subject.isInGroup ("wheel")) {
             return polkit.Result.YES;
         }
-});
\ No newline at end of file
+});
