From 0cfae760ea35c7f678ff7a289aef76c95fa6d9f4 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 11 Sep 2019 11:09:49 -0700
Subject: [PATCH 2/3] Add malloc_trim call.

---
 libdaemon/server/daemon-server.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libdaemon/server/daemon-server.c b/libdaemon/server/daemon-server.c
index bc58f7b..f7a343b 100644
--- a/libdaemon/server/daemon-server.c
+++ b/libdaemon/server/daemon-server.c
@@ -652,6 +652,7 @@ void daemon_start(daemon_state s)
 
 	while (!failed) {
 		_reset_timeout(s);
+		malloc_trim(0);
 		FD_ZERO(&in);
 		FD_SET(s.socket_fd, &in);
 
-- 
2.23.0

