diff --git a/neofetch b/neofetch
index 1e4b5646..42b8da10 100755
--- a/neofetch
+++ b/neofetch
@@ -794,8 +794,8 @@ image_source="auto"
 #       popos, Porteus, PostMarketOS, Proxmox, Puppy, PureOS, Qubes, Radix,
 #       Raspbian, Reborn_OS, Redstar, Redcore, Redhat, Refracted_Devuan,
 #       Regata, Rosa, sabotage, Sabayon, Sailfish, SalentOS, Scientific,
-#       Septor, SereneLinux, SharkLinux, Siduction, Slackware, SliTaz,
-#       SmartOS, Solus, Source_Mage, Sparky, Star, SteamOS, SunOS,
+#       Septor, SereneLinux, "Serpent OS", SharkLinux, Siduction, Slackware,
+#       SliTaz, SmartOS, Solus, Source_Mage, Sparky, Star, SteamOS, SunOS,
 #       openSUSE_Leap, openSUSE_Tumbleweed, openSUSE, SwagArch, Tails,
 #       Trisquel, Ubuntu-Budgie, Ubuntu-GNOME, Ubuntu-MATE, Ubuntu-Studio,
 #       Ubuntu, Venom, Void, Obarun, windows10, Windows7, Xubuntu, Zorin,
@@ -4936,12 +4936,12 @@ ASCII:
                                 Porteus, PostMarketOS, Proxmox, Puppy, PureOS, Qubes, Radix,
                                 Raspbian, Reborn_OS, Redstar, Redcore, Redhat, Refracted_Devuan,
                                 Regata, Rosa, sabotage, Sabayon, Sailfish, SalentOS, Scientific,
-                                Septor, SereneLinux, SharkLinux, Siduction, Slackware, SliTaz,
-                                SmartOS, Solus, Source_Mage, Sparky, Star, SteamOS, SunOS,
-                                openSUSE_Leap, openSUSE_Tumbleweed, openSUSE, SwagArch, Tails,
-                                Trisquel, Ubuntu-Budgie, Ubuntu-GNOME, Ubuntu-MATE, Ubuntu-Studio,
-                                Ubuntu, Venom, Void, Obarun, windows10, Windows7, Xubuntu, Zorin,
-                                and IRIX have ascii logos
+                                Septor, SereneLinux, \"Serpent OS\", SharkLinux, Siduction,
+                                Slackware, SliTaz, SmartOS, Solus, Source_Mage, Sparky, Star,
+                                SteamOS, SunOS, openSUSE_Leap, openSUSE_Tumbleweed, openSUSE,
+                                SwagArch, Tails, Trisquel, Ubuntu-Budgie, Ubuntu-GNOME, Ubuntu-MATE,
+                                Ubuntu-Studio, Ubuntu, Venom, Void, Obarun, windows10, Windows7,
+                                Xubuntu, Zorin and IRIX have ascii logos

                                 NOTE: Arch, Ubuntu, Redhat, and Dragonfly have 'old' logo variants.

@@ -9512,6 +9512,29 @@ ${c1}              __---''''''---__
             `''---______---''-``
 EOF
         ;;
+        "Serpent OS"*)
+            set_colors 7 2
+            read -rd '' ascii_data <<'EOF'
+${c1}            ,dKNWWNKOxo;.
+          ;xKXOdoloOWMWX0d:.
+        .dNNd'     cNMMMMNKOx;.
+       .dWNo       .:oxO000KXKl.
+       .OMNl             .....
+        oNMXd,.
+        .lKWMNKkdollccc:::;;;,'..
+          .;ox0KNWWMMMMMMMMWWWWNK0xl,.
+               ..,;:ccloodxO0KNWMMMMNOc.
+                             .';lxKWMMWk'
+      ..,:ccc:,.                  .oXMMMO
+   .:d0NWWWWWMWXOo,.     .,cc:;,,'..oWMMW
+ .c0WW0xl:;,;cd0NMNO:'..ldl;.    .'.:XMMM
+ xWMKc.        .;OWMWX0xl.         .dWMMK
+ WMWd.           .oKWMMXx,        .dNMMXc
+ NMMK:.      ..:oxo:oOXMMNOdc:;;cxKWMWO;
+ ;kXWN0xooodkOOko;.   .lONMMMMWWMMWKx;.
+   ,okKXXXKOxc'.        .:kKWMMWXOl'
+EOF
+        ;;

         "SharkLinux"*)
             set_colors 4 7
