From e4103526dc8808f77a7d18d38628539c01c23303 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <sunnyflunk@serpentos.com>
Date: Mon, 5 Jul 2021 11:55:10 +1000
Subject: [PATCH] Use modern linker locations for Serpent OS

Signed-off-by: Peter O'Connor <sunnyflunk@serpentos.com>
---
 gcc/config/i386/linux64.h | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/gcc/config/i386/linux64.h b/gcc/config/i386/linux64.h
index b3822ced5..6db616a17 100644
--- a/gcc/config/i386/linux64.h
+++ b/gcc/config/i386/linux64.h
@@ -27,13 +27,13 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 #define GNU_USER_LINK_EMULATION64 "elf_x86_64"
 #define GNU_USER_LINK_EMULATIONX32 "elf32_x86_64"
 
-#define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
-#define GLIBC_DYNAMIC_LINKERX32 "/libx32/ld-linux-x32.so.2"
+#define GLIBC_DYNAMIC_LINKER32 "/usr/lib32/ld-linux.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/usr/lib/ld-linux-x86-64.so.2"
+#define GLIBC_DYNAMIC_LINKERX32 "/usr/libx32/ld-linux-x32.so.2"
 
 #undef MUSL_DYNAMIC_LINKER32
-#define MUSL_DYNAMIC_LINKER32 "/lib/ld-musl-i386.so.1"
+#define MUSL_DYNAMIC_LINKER32 "/usr/lib32/ld-musl-i386.so.1"
 #undef MUSL_DYNAMIC_LINKER64
-#define MUSL_DYNAMIC_LINKER64 "/lib/ld-musl-x86_64.so.1"
+#define MUSL_DYNAMIC_LINKER64 "/usr/lib/ld-musl-x86_64.so.1"
 #undef MUSL_DYNAMIC_LINKERX32
-#define MUSL_DYNAMIC_LINKERX32 "/lib/ld-musl-x32.so.1"
+#define MUSL_DYNAMIC_LINKERX32 "/usr/libx32/ld-musl-x32.so.1"
-- 
2.31.1

