diff -Naur systemd-252.20221101/src/basic/locale-util.c systemd-252.20221101.tpg/src/basic/locale-util.c
--- systemd-252.20221101/src/basic/locale-util.c	2022-10-31 18:59:18.000000000 +0000
+++ systemd-252.20221101.tpg/src/basic/locale-util.c	2022-11-01 12:13:49.954372823 +0000
@@ -186,6 +186,31 @@
         return 0;
 }
 
+static int add_locales_from_sharedir(Set *locales) {
+        _cleanup_closedir_ DIR *dir = NULL;
+        int r;
+
+        dir = opendir("/usr/share/locale");
+        if (!dir)
+                return errno == ENOENT ? 0 : -errno;
+
+        FOREACH_DIRENT(de, dir, return -errno) {
+                char *z;
+
+                if (de->d_type != DT_DIR)
+                        continue;
+
+                z = normalize_locale(de->d_name);
+                if (!z)
+                        return -ENOMEM;
+
+                r = set_consume(locales, z);
+                if (r < 0 && r != -EEXIST)
+                        return r;
+        }
+
+        return 0;
+}
 int get_locales(char ***ret) {
         _cleanup_set_free_free_ Set *locales = NULL;
         _cleanup_strv_free_ char **l = NULL;
@@ -199,6 +224,10 @@
         if (r < 0 && r != -ENOENT)
                 return r;
 
+        r = add_locales_from_sharedir(locales);
+        if (r < 0)
+                return r;
+
         r = add_locales_from_libdir(locales);
         if (r < 0)
                 return r;
