From: Tomasz Buchert <tomasz@debian.org>
Date: Sun, 30 Aug 2020 12:45:51 +0200
Subject: remove missing test files reference

---
 python/tests/_test_utils.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/python/tests/_test_utils.py b/python/tests/_test_utils.py
index 104e654..2bdb091 100644
--- a/python/tests/_test_utils.py
+++ b/python/tests/_test_utils.py
@@ -36,10 +36,10 @@ TESTDATA_DIR = os.path.join(test_dir, 'testdata')
 
 TESTDATA_FILES = [
     'empty',  # Empty file
-    '10x10y',  # Small text
-    'alice29.txt',  # Large text
-    'random_org_10k.bin',  # Small data
-    'mapsdatazrh',  # Large data
+    # '10x10y',  # Small text
+    # 'alice29.txt',  # Large text
+    # 'random_org_10k.bin',  # Small data
+    # 'mapsdatazrh',  # Large data
 ]
 
 TESTDATA_PATHS = [os.path.join(TESTDATA_DIR, f) for f in TESTDATA_FILES]
