From a9e0715245d37003eae622a60056dcb622d2616a Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 31 Mar 2015 16:06:03 +0100
Subject: [PATCH 1/1] pam_env: Only report non ENOENT errors for env file

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 modules/pam_env/pam_env.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/modules/pam_env/pam_env.c b/modules/pam_env/pam_env.c
index 6f56ed3..7ae6e76 100644
--- a/modules/pam_env/pam_env.c
+++ b/modules/pam_env/pam_env.c
@@ -200,7 +200,9 @@ _parse_env_file(pam_handle_t *pamh, int ctrl, const char *file)
     D(("Env file name is: %s", file));
 
     if ((conf = fopen(file,"r")) == NULL) {
-      pam_syslog(pamh, LOG_ERR, "Unable to open env file: %s: %m", file);
+      if (errno != ENOENT) {
+	    pam_syslog(pamh, LOG_ERR, "Unable to open env file: %s: %m", file);
+      }
       return PAM_IGNORE;
     }
 
-- 
2.35.4

