From 67b19165f2631cbad0c39ec14ac0b092971cb38e Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 17 Dec 2022 20:21:16 -0600
Subject: [PATCH] Stateless pam_time config file

---
 modules/pam_time/pam_time.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/modules/pam_time/pam_time.c b/modules/pam_time/pam_time.c
index 089ae22..72d7547 100644
--- a/modules/pam_time/pam_time.c
+++ b/modules/pam_time/pam_time.c
@@ -39,6 +39,8 @@
 #define PAM_DEBUG_ARG       0x0001
 #define PAM_NO_AUDIT        0x0002
 
+#define DEFAULT_PAM_TIME_CONF  "/usr/share/defaults/etc/security/time.conf"
+
 #ifndef TRUE
 # define TRUE 1
 #endif
@@ -53,7 +55,11 @@ _pam_parse (const pam_handle_t *pamh, int argc, const char **argv, const char **
 {
     int ctrl = 0;
 
-    *conffile = PAM_TIME_CONF;
+	if (access(PAM_TIME_CONF, F_OK) == 0)
+		*conffile = PAM_TIME_CONF;
+	else
+		*conffile = DEFAULT_PAM_TIME_CONF;
+
     /* step through arguments */
     for (; argc-- > 0; ++argv) {
 	const char *str;
-- 
2.38.1

