From ecb6fc1e8fe00119f006cd9806ba6d3c4876be09 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 17 Dec 2022 20:29:56 -0600
Subject: [PATCH] Stateless pam_limits config file

---
 modules/pam_limits/pam_limits.c | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/modules/pam_limits/pam_limits.c b/modules/pam_limits/pam_limits.c
index 7cc45d7..81f80d4 100644
--- a/modules/pam_limits/pam_limits.c
+++ b/modules/pam_limits/pam_limits.c
@@ -123,6 +123,8 @@ struct pam_limit_s {
 
 #define CONF_FILE (pl->conf_file != NULL)?pl->conf_file:LIMITS_FILE
 
+#define DEFAULT_CONF_FILE "/usr/share/defaults/etc/security/limits.conf"
+
 static int
 _pam_parse (const pam_handle_t *pamh, int argc, const char **argv,
 	    struct pam_limit_s *pl)
@@ -814,7 +816,12 @@ parse_config_file(pam_handle_t *pamh, const char *uname, uid_t uid, gid_t gid,
     /* check for the LIMITS_FILE */
     if (ctrl & PAM_DEBUG_ARG)
         pam_syslog(pamh, LOG_DEBUG, "reading settings from '%s'", CONF_FILE);
-    fil = fopen(CONF_FILE, "r");
+
+    if (access(CONF_FILE, F_OK) == 0)
+        fil = fopen(CONF_FILE, "r");
+    else
+        fil = fopen(DEFAULT_CONF_FILE, "r");
+
     if (fil == NULL) {
         pam_syslog (pamh, LOG_WARNING,
 		    "cannot read settings from %s: %m", CONF_FILE);
-- 
2.38.1

