From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Wed, 29 Jan 2025 19:39:56 -0600
Subject: [PATCH] pkgconfig: Require sdl2_compat

---
 SDL2_mixer.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SDL2_mixer.pc.in b/SDL2_mixer.pc.in
index c14ffb6..b0df72f 100644
--- a/SDL2_mixer.pc.in
+++ b/SDL2_mixer.pc.in
@@ -6,7 +6,7 @@ includedir=@includedir@
 Name: SDL2_mixer
 Description: mixer library for Simple DirectMedia Layer
 Version: @VERSION@
-Requires: sdl2 >= @SDL_VERSION@
+Requires: sdl2_compat >= @SDL_VERSION@
 Libs: -L${libdir} -lSDL2_mixer
 Cflags: -I${includedir}/SDL2
 Requires.private: @PC_REQUIRES@
