--- binutils-2.39/gas/as.c~	2022-07-08 09:46:47.000000000 +0000
+++ binutils-2.39/gas/as.c	2022-11-06 19:04:06.769936497 +0000
@@ -1273,13 +1273,6 @@
   start_time = get_run_time ();
   signal_init ();
 
-#ifdef HAVE_LC_MESSAGES
-  setlocale (LC_MESSAGES, "");
-#endif
-  setlocale (LC_CTYPE, "");
-  bindtextdomain (PACKAGE, LOCALEDIR);
-  textdomain (PACKAGE);
-
   if (debug_memory)
     chunksize = 64;
 
--- binutils-2.39/gas/input-file.c~	2022-07-08 09:46:47.000000000 +0000
+++ binutils-2.39/gas/input-file.c	2022-11-06 19:07:49.762259813 +0000
@@ -125,7 +125,7 @@
   gas_assert (filename != 0);	/* Filename may not be NULL.  */
   if (filename[0])
     {
-      f_in = fopen (filename, FOPEN_RT);
+      f_in = fopen (filename, "rm");
       file_name = filename;
     }
   else
@@ -202,6 +202,8 @@
 void
 input_file_close (void)
 {
+  /* For performance, just leak the FD, we exit instantly anyway */
+  return;
   /* Don't close a null file pointer.  */
   if (f_in != NULL)
     fclose (f_in);
--- binutils-2.39/gas/input-file.c~	2022-11-06 19:07:49.000000000 +0000
+++ binutils-2.39/gas/input-file.c	2022-11-06 20:19:49.353159757 +0000
@@ -250,8 +250,6 @@
     return_value = where + size;
   else
     {
-      if (fclose (f_in))
-	as_warn (_("can't close %s: %s"), file_name, xstrerror (errno));
 
       f_in = (FILE *) 0;
       return_value = 0;
