From 34e59dca1d6ddcb1db92ee53de87cbeab35f8613 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@serpentos.com>
Date: Fri, 22 Nov 2024 21:57:37 +0000
Subject: [PATCH] Set plymouth defaults

Signed-off-by: Ikey Doherty <ikey@serpentos.com>
---
 src/plymouthd.defaults | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plymouthd.defaults b/src/plymouthd.defaults
index 7f247fd..48ab9fa 100644
--- a/src/plymouthd.defaults
+++ b/src/plymouthd.defaults
@@ -1,6 +1,6 @@
 # Distribution defaults. Changes to this file will get overwritten during
 # upgrades.
 [Daemon]
-Theme=spinner
+Theme=bgrt
 ShowDelay=0
 DeviceTimeout=8
-- 
2.46.0

