From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <sunnyflunk@gmail.com>
Date: Sun, 30 Oct 2016 17:03:15 +1100
Subject: [PATCH] Support stateless config directories

---
 src/corelib/io/qstandardpaths_unix.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/corelib/io/qstandardpaths_unix.cpp b/src/corelib/io/qstandardpaths_unix.cpp
index 4ebeefcedb..f26fa3427c 100644
--- a/src/corelib/io/qstandardpaths_unix.cpp
+++ b/src/corelib/io/qstandardpaths_unix.cpp
@@ -379,7 +379,11 @@ static QStringList xdgConfigDirs()
     // http://standards.freedesktop.org/basedir-spec/latest/
     const QString xdgConfigDirs = QFile::decodeName(qgetenv("XDG_CONFIG_DIRS"));
     if (xdgConfigDirs.isEmpty())
+    {
+        dirs.append(QString::fromLatin1("/run/xdg"));
         dirs.append(QString::fromLatin1("/etc/xdg"));
+        dirs.append(QString::fromLatin1("/usr/share/xdg"));
+    }
     else
         dirs = xdgConfigDirs.split(QLatin1Char(':'));
     return dirs;
