From 9b7c7084c9ac997768bab4d3f22ee72b522104fc Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 24 Sep 2022 18:58:48 -0500
Subject: [PATCH 1/1] Stateless pam_env config file

---
 modules/pam_env/pam_env.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/modules/pam_env/pam_env.c b/modules/pam_env/pam_env.c
index 7ae6e76..4fdb34b 100644
--- a/modules/pam_env/pam_env.c
+++ b/modules/pam_env/pam_env.c
@@ -13,6 +13,8 @@
 #define DEFAULT_USER_ENVFILE    ".pam_environment"
 #define DEFAULT_USER_READ_ENVFILE 0
 
+#define DEFAULT_CONF_STATELESS_FILE     "/usr/share/defaults/etc/security/"
+
 #include "config.h"
 
 #include <ctype.h>
@@ -85,7 +87,11 @@ _pam_parse (const pam_handle_t *pamh, int argc, const char **argv,
 
     *readenv = DEFAULT_READ_ENVFILE;
     *user_readenv = DEFAULT_USER_READ_ENVFILE;
-    *conffile = DEFAULT_CONF_FILE;
+
+    if (access(DEFAULT_CONF_FILE, F_OK) == 0)
+        *conffile = DEFAULT_CONF_FILE;
+    else
+        *conffile = DEFAULT_CONF_STATELESS_FILE;
 
     /* step through arguments */
     for (; argc-- > 0; ++argv) {
-- 
2.35.4

