From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <sunnyflunk@serpentos.com>
Date: Mon, 5 Jul 2021 14:22:14 +1000
Subject: [PATCH] Make gnu hash the default for lld and clang

Origin: vendor
Forwarded: not-needed
Last-Update: 2025-01-14
---
 lld/ELF/Driver.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lld/ELF/Driver.cpp b/lld/ELF/Driver.cpp
index 47393e6823b3..bcc7971042c6 100644
--- a/lld/ELF/Driver.cpp
+++ b/lld/ELF/Driver.cpp
@@ -1859,7 +1859,7 @@ static void setConfigs(opt::InputArgList &args) {
     if (config->emachine == EM_MIPS)
       config->sysvHash = true;
     else
-      config->sysvHash = config->gnuHash = true;
+      config->gnuHash = true;
   }
 
   // Set default entry point and output file if not specified by command line or
