diff -Naur systemd-252.20221101/src/libsystemd/sd-event/sd-event.c systemd-252.20221101.tpg/src/libsystemd/sd-event/sd-event.c
--- systemd-252.20221101/src/libsystemd/sd-event/sd-event.c	2022-10-31 18:59:18.000000000 +0000
+++ systemd-252.20221101.tpg/src/libsystemd/sd-event/sd-event.c	2022-11-01 13:13:52.028969974 +0000
@@ -3,6 +3,7 @@
 #include <sys/epoll.h>
 #include <sys/timerfd.h>
 #include <sys/wait.h>
+#include <malloc.h>
 
 #include "sd-daemon.h"
 #include "sd-event.h"
@@ -4295,9 +4296,18 @@
         PROTECT_EVENT(e);
 
         r = sd_event_prepare(e);
-        if (r == 0)
+        if (r == 0) {
+                /* There was nothing? Then wait...
+                *
+                * ... but since there is no event pending, likely
+                * we'll be idle for a long time.  This is a good time
+                * to return any malloc() memory reserves to the
+                * operating system.
+                */
+               malloc_trim(0);
                 /* There was nothing? Then wait... */
                 r = sd_event_wait(e, timeout);
+        }
 
         if (e->profile_delays)
                 e->last_run_usec = now(CLOCK_MONOTONIC);
