diff -up systemd-246.20200806/units/systemd-random-seed.service.in.11~ systemd-246.20200806/units/systemd-random-seed.service.in
--- systemd-246.20200806/units/systemd-random-seed.service.in.11~	2020-08-06 22:58:05.682295792 +0200
+++ systemd-246.20200806/units/systemd-random-seed.service.in	2020-08-06 22:59:04.525054215 +0200
@@ -27,4 +27,6 @@ ExecStop=@rootlibexecdir@/systemd-random
 # used as ordering barrier for service that require an initialized entropy
 # pool. Since initialization can take a while on entropy-starved systems, let's
 # increase the timeout substantially here.
-TimeoutSec=10min
+# But since a typical user will power-cycle if booting takes 10 minutes
+# (which will just make things worse), let's not slow down bootup too badly.
+TimeoutSec=10sec
