diff --git a/moss/build.rs b/moss/build.rs
index e049882..7f5311b 100644
--- a/moss/build.rs
+++ b/moss/build.rs
@@ -2,5 +2,5 @@ extern crate varlink_generator;
 
 fn main() {
     println!("cargo:rerun-if-changed=src/varlink/com.serpentos.moss.varlink");
-    varlink_generator::cargo_build_tosource("src/varlink/com.serpentos.moss.varlink", true);
+    varlink_generator::cargo_build_tosource("src/varlink/com.serpentos.moss.varlink", false);
 }
