From 592d916251b3777166134da3eebe711d10a1404c Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@serpentos.com>
Date: Wed, 7 Aug 2024 23:46:47 +0000
Subject: [PATCH] Use vendor-OS PAM configuration only

Signed-off-by: Ikey Doherty <ikey@serpentos.com>
---
 greetd/src/server.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/greetd/src/server.rs b/greetd/src/server.rs
index b215c4a..f49408a 100644
--- a/greetd/src/server.rs
+++ b/greetd/src/server.rs
@@ -198,7 +198,7 @@ impl Drop for Listener {
 }
 
 pub async fn main(config: Config) -> Result<(), Error> {
-    let service = if Path::new(&format!("/etc/pam.d/{}", config.file.general.service)).exists() {
+    let service = if Path::new(&format!("/usr/share/defaults/etc/pam.d/{}", config.file.general.service)).exists() {
         &config.file.general.service
     } else if Path::new(&format!("/usr/lib/pam.d/{}", config.file.general.service)).exists() {
         &config.file.general.service
@@ -207,7 +207,7 @@ pub async fn main(config: Config) -> Result<(), Error> {
     };
 
     let greeter_service = if Path::new(&format!(
-        "/etc/pam.d/{}",
+        "/usr/share/defaults/etc/pam.d/{}",
         config.file.default_session.service
     ))
     .exists()
-- 
2.46.0

