Description: move ld.so.cache to /var/cache/ldconfig
===================================================================
Index: glibc-2.21/sysdeps/generic/dl-cache.h
===================================================================
--- glibc-2.21.orig/sysdeps/generic/dl-cache.h	2015-02-06 06:40:18.000000000 +0000
+++ glibc-2.21/sysdeps/generic/dl-cache.h	2015-03-10 10:01:19.867830249 +0000
@@ -28,7 +28,7 @@
 #endif
 
 #ifndef LD_SO_CACHE
-# define LD_SO_CACHE SYSCONFDIR "/ld.so.cache"
+# define LD_SO_CACHE "/var/cache/ldconfig/ld.so.cache"
 #endif
 
 #ifndef add_system_dir
